indico/modules/designer/placeholders.py

```diff
@@ -71,6 +75,22 @@ def render(cls, registration):
         return name
 
 
+class EventLogoPlaceholder(DesignerPlaceholder):
+    group = 'event'
+    name = 'event_logo'
+    description = _("Event Logo")
+
+    @classmethod
+    def render(cls, registration):
+        try:
+            buf = io.BytesIO(registration.logo)
+            img = Image.open(buf)
+            return img
```

Comment:
```suggestion
            buf = io.BytesIO(registration.logo)
            return Image.open(buf)
```
or even
```suggestion
            return Image.open(io.BytesIO(registration.logo))
```
(though in my opinion it feels less cluttered with the first option)